badge_server/test_badges.py

```diff
@@ -31,72 +31,216 @@
     [package.Package('apache-beam[gcp]')],
     python_major_version=2,
     status=compatibility_store.Status.SUCCESS,
+    dependency_info={
+        'apache-beam[gcp]': {
+            'current_time': datetime.datetime.utcnow(),
+            'installed_version': '2.12.0',
+            'installed_version_time': datetime.datetime(
+                2019, 2, 19, 21, 15, 56),
+            'is_latest': True,
+            'latest_version': '2.12.0',
+            'latest_version_time': datetime.datetime(
+                2019, 2, 19, 21, 15, 56),
+        },
+    },
     timestamp=datetime.datetime.utcnow())
 
 APACHE_BEAM_GIT_RECENT_SUCCESS_2 = compatibility_store.CompatibilityResult(
     [package.Package('git+git://github.com/google/apache-beam.git')],
     python_major_version=2,
     status=compatibility_store.Status.SUCCESS,
+    dependency_info={
+        'apache-beam[gcp]': {
+            'current_time': datetime.datetime.utcnow(),
+            'installed_version': '2.12.0',
+            'installed_version_time': datetime.datetime(
+                2019, 2, 19, 21, 15, 56),
+            'is_latest': True,
+            'latest_version': '2.12.0',
+            'latest_version_time': datetime.datetime(
+                2019, 2, 19, 21, 15, 56),
+        },
+    },
     timestamp=datetime.datetime.utcnow())
 
 GOOGLE_API_CORE_RECENT_SUCCESS_2 = compatibility_store.CompatibilityResult(
     [package.Package('google-api-core')],
     python_major_version=2,
     status=compatibility_store.Status.SUCCESS,
+    dependency_info={
+        'google-api-core': {
+            'current_time': datetime.datetime.utcnow(),
```

Comment:
Is there a reason why this has to be "now"? This means that every time the test is run, it will be run with different input, which means it will be hard to reproduce. I see we've already been using this pattern, e.g. on old line 46.